### PR TITLE
[Feature] Fastboot Boot (Draft)

### PR DESCRIPTION
**Pull request by newtonick**
Draft - Work in Progress

- Addition of pi0X and pi0X-dev. These are duplicates of pi0 and pi0-dev but "Experimental". I'll merge the changes into pi0 and pi0-dev once my testing is done and remove pi0X and pi0X-dev.
- Updates to build.sh script
	- replace --keep-alive and --skip-build with --no-op
- buildroot upgrade/update to 2023.02 tag 8cca1e6de1c69a0a5e876116906bb3f6da4a5bd5
- remove python-qrcode external package and move to internal buildroot package
- remove S02mdev overlay, not needed
- remove modprobe from S02seedsigner
- overlay for S10mdev to remove modprobe loads
- overlay for fstab to remove tmpfs

### CODE REVIEW

**Comment by jdlcdl** [comment]
I gave this a shot on debian w/o: `./build.sh --pi0X` and `./build.sh --pi0X --dev` to no avail.

The images did successfully get built.
It seems like my pi0 tries to boot (green led and red camera led are active in the first few seconds) but then it never reaches the seedsigner app.  I don't have connectors for watching what might be on the console in the dev version, so other than "it didn't work", I cannot add more.

The image you prepared and shared on telegram the other day worked great.  I saw the logo in 17s and I was in the main menu in 24s.

File: opt/build.sh [hunk 1]
```diff
@@ -17,7 +17,9 @@ help()
       --pi02w       Build for pi02w and pi3
       --pi4         Build for pi4 and pi4cmio
       --pi0X        Experimental build for pi0
+      --pi2         Experimental build for pi2
```
Comment (jdlcdl) [inline opt/build.sh:20]: `--pi2` becomes `--pi2X`

**Comment by jdlcdl** [comment]
> pi2X working

I confirm!  12s to screen initialization, 19s to main menu.

Nice!

**Comment by jdlcdl** [comment]
From the perspective of an Intel Xeon based Ubuntu system using docker-compose v1.29.2 and docker v20.10.21:

as of commit dbfe0e3, up to 2723caa, I'm consistently reproducing:
* pi0X images w/sha256: 31e1532aacee8ff474a2a88db572203dc18df0ae1b14149fa65bcdc936932a97
* pi0X-dev images w/sha256: 81df9ef373923e7be2242e6c217daf946beebb2f6cebed6a8fd2cec2888de338
...BUT only when I've specified to build each image as the first one built in a new container.

I can quickly build the `--pi0X --dev --no-clean` version if I've already built `--pi0X` inside the container, and the image boots fine, but sha256sums seem to drift once container has already been used for another version, unless rebuilding what you last built.  Same goes for building the `--pi0X --no-clean` version after a `--pi0X --dev` version had been built, sha256sums will differ than the same image built in a fresh container.

**Comment by bitcoinprecept** [comment]
This item has been added to the Seedsigner Feature Development Board

**Pull request closed without merging**